web-app/js/portal/ui/MainPanel.js

```diff
@@ -46,15 +46,15 @@ Portal.ui.MainPanel = Ext.extend(Ext.Panel, {
 
         Portal.ui.MainPanel.superclass.constructor.call(this, config);
 
-        Ext.MsgBus.subscribe(PORTAL_EVENTS.VIEW_DATA_COLLECTION, this._onViewGeoNetworkRecord, this);
-        Ext.MsgBus.subscribe(PORTAL_EVENTS.DATA_COLLECTION_ADDED, this._onGeoNetworkRecordAdded, this);
+        Ext.MsgBus.subscribe(PORTAL_EVENTS.VIEW_DATA_COLLECTION, this._onViewMetadataRecord, this);
```

Comment:
Re `VIEW_DATA_COLLECTION` - usually, event are to indicate that something just happened, not a command to do something, such as this one.  If it wasn't for the multi-select functionality, I think this event could go - with the `MainPanel` simply electing to switch to step 2 on `DATA_COLLECTION_ADDED`.

Maybe there's some other way we could do this - thoughts (cc @dnahodil)?